src/api/hooks/useGetANS.ts

```diff
@@ -59,3 +59,26 @@ export function useGetAddressFromName(name: string) {
 
   return address;
 }
+
+export async function getAddressFromName(
+  name: string,
+  network: NetworkName,
+): Promise<string | undefined> {
+  const addressUrl = getFetchAddressUrl(network, name);
+  if (addressUrl === undefined) {
+    return undefined;
+  }
+
+  const addressResponse = await fetch(addressUrl);
```

Comment:
should we add `catch`?

Reply:
Hmm... I can't think of a case when it'll throw error, unless the url has changed.

Reply:
Oh and the upper level where this function is called has try catch. So it should be safe.

Reply:
if the server is down for some reason...?